package.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "web",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
+    "@fortawesome/free-brands-svg-icons": "^5.14.0",
+    "@fortawesome/free-regular-svg-icons": "^5.14.0",
+    "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.11",
+    "@onaio/connected-private-route": "^0.0.11",
+    "@onaio/connected-reducer-registry": "^0.0.3",
+    "@onaio/gatekeeper": "^0.0.21",
+    "@onaio/redux-reducer-registry": "^0.0.9",
+    "@onaio/session-reducer": "^0.0.11",
+    "@onaio/utils": "^0.0.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/bootstrap": "^4.5.0",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "@types/react-helmet": "^6.1.0",
+    "@types/react-redux": "^7.1.9",
+    "@types/react-router": "^5.1.8",
+    "@types/react-router-dom": "^5.1.5",
+    "@types/reactstrap": "^8.5.1",
+    "@types/redux-mock-store": "^1.0.2",
+    "@types/redux-testkit": "^1.0.5",
+    "@types/seamless-immutable": "^7.1.13",
```

Comment:
Would be cool to move these to devDependencies

Reply:
addressed https://github.com/OpenSRP/web/pull/9/commits/032c2883cc5cf8acd89ccdbf9b17c71a0b63e239